src/imitation/scripts/config/train_adversarial.py

```diff
@@ -59,6 +62,18 @@ def train_defaults():
   rollout_hint = None  # Used to generate default rollout_path
 
 
+@train_ex.config
+def timesteps(n_epochs, batch_size):
```

Comment:
I think it will be useful in hp tuning to be able to change the batch size while keeping the number of updates (via n_epochs) the same. Making a change right now to allow specifying either `total_timesteps` or `n_epochs`.

Reply:
Sorry for slow response. This seems odd to me: algorithms will almost always do better with more `total_timesteps`. Normally you choose some fixed `total_timesteps` budget, and search for hyperparameters that do best given that budget.

Reply:
OK, I'll get rid of the `n_epochs` option